netlify.toml

```diff
@@ -21,6 +21,7 @@
   [context.production.environment]
     REACT_APP_NETWORK = "mainnet"
     REACT_APP_GOOGLE_ANALYTICS = "GTM-KTSX2PL"
+    REACT_APP_GOOGLE_ANALYTICS_CLICKTHROUGH = "GTM-5MWCJN3"
```

Comment:
Why is this different than in local env files?

Reply:
thanks, forgot to copy over correct id there - updated now